Playbooks/Create Incidents with Email/Readme.md

```diff
@@ -0,0 +1,46 @@
+
+
+
+
+Please use the below button to deploy
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FCreate%2520Incidents%2520with%2520Email%2Fazuredeploy.json)
+
+### Deployment
+
+Once deployed you can configure the connetions as below
+
+![image](https://user-images.githubusercontent.com/20562985/196198346-2dbde9ca-9812-4c49-a0e6-d3ebca777bb3.png)
+
+
+### Configuration
+
+1. Sentinel Connection
+
```

Comment:
Screenshot is good , but it will look nicer if you can provide the step by step approach to create and configured the connection

Reply:
Generally connections are configured during the ARM template deployment.
This is a fallback step added to ensure the tool does not break.
Furthermore, if this logic app is expected to be configured as solution, this separate configuration will not be necessary.
However, in case you suggest this as a mandatory step for the PR merge, please provide an example and will be able to update the PR.
Thank you & apologies for being late in the response.